### PR TITLE
Smoke: Separated scripts for Register and UnregisterAppInterface from smoke API script

### PR DESCRIPTION
**Pull request by GetmanetsIrina**
Separated scripts for RegisterAppInterface and UnregisterAppInterface from smoke API script

### CODE REVIEW

File: test_scripts/Smoke/API/037_RegisterAppInterface_PositiveCase.lua [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+--[[ Scenario ]]
+runner.Title("Preconditions")
+runner.Step("Clean environment", commonSmoke.preconditions)
+runner.Step("Start SDL, HMI, connect Mobile, start Session", commonSmoke.start)
+runner.Step("RAI, PTU", commonSmoke.registerApplicationWithPTU)
+runner.Step("Activate App", commonSmoke.activateApp)
+runner.Step("UnregisterAppInterface Positive Case", unregisterAppInterface)
+
+runner.Title("Test")
+runner.Step("RegisterAppInterface Positive Case", RegisterAppInterface, { allParams })
```
Comment (dboltovskyi) [inline test_scripts/Smoke/API/037_RegisterAppInterface_PositiveCase.lua]: @GetmanetsIrina Think we can omit paramter ```allParams```` here.
It's possible to use local variables and functions directly in  ```RegisterAppInterface()``` function

File: test_scripts/Smoke/API/037_RegisterAppInterface_PositiveCase.lua [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+		transportType = "WIFI",
+		isSDLAllowed = true
+	}
+	notificationParams.application.policyAppID = requestParams.appID
+	notificationParams.ttsName = requestParams.ttsName
+	notificationParams.vrSynonyms = requestParams.vrSynonyms
+	return notificationParams
+end
+
+local allParams = {
```
Comment (dboltovskyi) [inline test_scripts/Smoke/API/037_RegisterAppInterface_PositiveCase.lua]: @GetmanetsIrina Bearing in mind the last comment from my side we can omit defining of ```allPArams``` variable here and to create 2 ones: ```requestParams``` and ```notificationParams```
This simplify code a little

File: test_scripts/Smoke/API/036_UnregisterAppInterface_PositiveCase.lua [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+---------------------------------------------------------------------------------------------------
+
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local commonSmoke = require('test_scripts/Smoke/commonSmoke')
+
+--[[ Local Functions ]]
+local function unregisterAppInterface(self)
+	local cid = self.mobileSession1:SendRPC("UnregisterAppInterface", { })
+	EXPECT_HMINOTIFICATION("BasicCommunication.OnAppUnregistered", { appID = self.appID, unexpectedDisconnect = false })
```
Comment (dboltovskyi) [inline test_scripts/Smoke/API/036_UnregisterAppInterface_PositiveCase.lua]: @GetmanetsIrina Currently ```self.appID``` is nill. It should be used ```getHMIAppId()``` function from common module instead.

Comment (GetmanetsIrina) [reply test_scripts/Smoke/API/036_UnregisterAppInterface_PositiveCase.lua]: @dboltovskyi , Done in 607c20fb6767f2c4c0e34e662869ef978977e810.

File: test_scripts/Smoke/API/037_RegisterAppInterface_PositiveCase.lua (hunk 2 above)
Comment (GetmanetsIrina) [reply test_scripts/Smoke/API/037_RegisterAppInterface_PositiveCase.lua]: @dboltovskyi , Done in 607c20fb6767f2c4c0e34e662869ef978977e810.

File: test_scripts/Smoke/API/037_RegisterAppInterface_PositiveCase.lua (hunk 1 above)
Comment (GetmanetsIrina) [reply test_scripts/Smoke/API/037_RegisterAppInterface_PositiveCase.lua]: @dboltovskyi , Done in 607c20fb6767f2c4c0e34e662869ef978977e810.

**Pull request merged**